content/en/load_balancer.md

```diff
@@ -5,10 +5,14 @@ category: concept
 ---
 
 ## What it is
-A load balancer is a method to distribute incoming network traffic across a group of servers in the back-end. The solution can be software or hardware based.
+
+A load balancer is a way to distribute incoming user requests among multiple instances of an application. The solution can be software or hardware based.
 
 ## Problem it addresses
-This helps solve the issue related to high availability and distributed systems. When working on an application or service that needs to scale to hundreds of thousands of users, one will often need to distribute that application on multiple servers. The load balancer is the "traffic cop" that routes traffic.
+
+Suppose a person starts a business and builds a website. Initially, the number of people visiting their site would be less hence the speed with which the server(a computer that sends some information in response to a user’s request) serves the website would be optimal. Now the person invests in some marketing and their business starts to grow. This would result in an increased number of users visiting their website (probably from all around the globe). One server might not be enough to handle the increased number of requests. And would result in a slower website and frequent crashing of the server. One would have to add more servers to handle these many requests. But what if one of the servers goes down, what happens to the users it was serving? There was no way of keeping a check on these situations.
```

Comment:
This example is making a case for replicas, not a load balancer. Without a load balancer, all requests might be routed to the same service while other services are available. 

Also, please keep the examples short and sweet. A person building a website...people visiting the website...investing in marketing...business grows... that are too many details that are quite distracting.

Reply:
Yes I'll make the changes and update.